_posts/2021-02-21-goodbye-scrum.md

```diff
@@ -0,0 +1,79 @@
+
+The IDEA will contain items that are SWAGged, i.e. they'll contain just enough information that we were able to come up with a SWAG. This helps us have ROI conversations that inform the priority of these items.
+
+From IDEA, items move into SPECIFY. These are essentially on-deck items. This is where (and when) on-demand planning happens. When a SPECIFY item is picked up, the owner's responsibility is to break it down into max 2-3 day tickets in the TODO column. The result is a more detailed plan. But the TODO column does have a WIP limit. If a SPECIFY task's decomposition is too big to fit into the TODO column, then break the SPECIFY task apart in-place in order to adhere to our limits by specifying smaller chunks of better-understood work.
+
+The items in TODO are from where new work is picked up. When you're ready to pull a new task, grab the topmost ticket from TODO that you've got the skills to do. (This is where we can inject escalated bugs, etc.) Or, better yet, pick up the top task — and if you're not sure how to do it, pair with somebody that does!
+
+When picking up an item from TODO, you'll move it into DEV. If you cannot because there's already too many tasks in that column, you pair/mob!
+
+Tasks then move from DEV into VERIFY. This could be a simple deploy/test step, or it could be a more complex wire-up-all-the-things step. At this stage of the lifecycle it's about shipping something to the customer (and so still valuable for us to be pairing/mobbing). Finally, once you've verified it's all good, drag to DONE.
+
+The final piece of the puzzle are the WIP limits. Using the Theory of Constraints, we can probably calculate the ideal WIP limits (any takers?!) but I think we'll likely trial-and-error our way to what works for us and strikes a good balance between throughput and pairing/mobbing.
+
+Now, for the hard part. For this to work, there's a few things that we need to be disciplined about:
+
+1. Everything we're working on must be represented in Jira. We're not doing Kanban if we're not using WIP limits, and WIP limits are irrelevant if we're doing stuff that's not tracked in Jira.
+2. Tasks in Jira need to be small (2-3 days max for starters). Large tasks prevent us from visualizing the progression of work, which will lead to a feeling of projects being stalled.
+
+Initially we'll take a MITA (Mike Is The Asshole) approach to ensuring the above happens. Please feel free to blame me for any discomfort or awkwardness. e.g. "Sorry, Mike is an asshole and is making me pair w/ you" or "Ugh I wish I could start that task, but Mike is being an asshole" and also "Mike's an asshole and we need to break this down before we continue working on it." But believe it or not, I do find it exhausting to be an asshole. So, I'd like everyone to take part in ensuring we're being disciplined about how we implement Kanban.
```

Comment:
This paragraph is also talking to the people you manage rather than project managers as a whole. I think it could be improved to make the audience less targeted.